djlsp/constants.py

```diff
@@ -1,113 +1,147 @@
-import re
+BUILTIN = "__builtins__"
 
-TAGS = [
-    "autoescape",
-    "block",
-    "comment",
-    "filter",
-    "for",
-    "if",
-    "ifchanged",
-    "spaceless",
-    "verbatim",
-    "with",
-    # TODO: check if i18n is loaded
-    "blocktranslate",
-    # TODO: check if l10n is loaded
-    "localize",
-    # TODO: check if tz is loaded
-    "localtime",
-    "timezone",
-]
-
-SIMPLE_TAGS = [
-    "csrf_token",
-    "cycle",
-    "debug",
-    "extends",
-    "firstof",
-    "include",
-    "load",
-    "lorem",
-    "now",
-    "regroup",
-    "resetcycle",
-    "templatetag",
-    "url",
-    "widthratio",
-    # TODO: check if statis is loaded
-    "static",
-    # TODO: check if i18n is loaded
-    "translate",
-    # TODO: Hack to make empy and else work, should check if inside if or for
-    "empty",
-    "else",
-]
-
-FILTERS = [
-    "add",
-    "addslashes",
-    "capfirst",
-    "center",
-    "cut",
-    "date",
-    "default",
-    "default_if_none",
-    "dictsort",
-    "dictsortreversed",
-    "divisibleby",
-    "escape",
-    "escapejs",
-    "escapeseq",
-    "filesizeformat",
-    "first",
-    "floatformat",
-    "force_escape",
-    "get_digit",
-    "iriencode",
-    "join",
-    "json_script",
-    "last",
-    "length",
-    "length_is",
-    "linebreaks",
-    "linebreaksbr",
-    "linenumbers",
-    "ljust",
-    "lower",
-    "make_list",
-    "phone2numeric",
-    "pluralize",
-    "pprint",
-    "random",
-    "rjust",
-    "safe",
-    "safeseq",
-    "slice",
-    "slugify",
-    "stringformat",
-    "striptags",
-    "time",
-    "timesince",
-    "timeuntil",
-    "title",
-    "truncatechars",
-    "truncatechars_html",
-    "truncatewords",
-    "truncatewords_html",
-    "unordered_list",
-    "upper",
-    "urlencode",
-    "urlize",
-    "urlizetrunc",
-    "wordcount",
-    "wordwrap",
-    "yesno",
-    # TODO: check if l10n is loaded
-    "localize",
-    "unlocalize",
-    # TODO: check if tz is loaded
-    "localtime",
-    "utc",
-    "timezone",
-]
+FALLBACK_DJANGO_DATA = {
+    "static_files": [],
+    "urls": [],
+    "templates": [],
+    "libraries": {
+        "__builtins__": {
```

Comment:
Je zou hier de BUILTIN constante kunnen gebruiken. Zo niet dan kan deze volgens mij weg want hij wordt hier nergens anders gebruikt zover ik kan zien.

Reply:
Ik zie in parser.py dat deze gebruikt wordt dus deze comment hierboven komt te vervallen

Reply:
`BUILTIN` is used in parser, I didn't use it in the fallback because this is a dump of empty django project